package-lock.json

```diff
@@ -5409,7 +5409,7 @@
     },
     "expand-range": {
       "version": "1.8.2",
-      "resolved": "https://registry.npmjs.org/expand-range/-/expand-range-1.8.2.tgz",
+      "resolved": "http://registry.npmjs.org/expand-range/-/expand-range-1.8.2.tgz",
```

Comment:
This is weird, can you remove the package-lock changes from this PR please?

Reply:
yeah, it's weird -might be worth looking into why my npm is doing that - will remove the package-lock changes from this PR